build.rs

```diff
@@ -2,39 +2,48 @@ fn main() {
     #[cfg(feature = "std")]
     compile_rpo_falcon();
 
-    #[cfg(feature = "arch-arm64-sve")]
+    #[cfg(all(target_feature = "sve", feature = "sve"))]
```

Comment:
would it make sense to remove the feature `sve` and rely exclusively on `target_feature`?

Reply:
I wanted to have an option to compile without SVE acceleration even on platforms which support SVE. The motivation is that there might be some unforeseen issues on platforms we haven't tested, and so if the compilation with SVE fails, would be good to be able to compile the code without SVE.